Assets/General/Scripts/MenuFunctionality/PauseMenu.cs

```diff
@@ -34,7 +34,11 @@ private void Awake()
     /// </summary>
     public void PauseToggle()
     {
-        TimeManager.Instance.PauseGameToggle();
+        //don't unpause if the settings scene is loaded
+        if (AframaxSceneManager.Instance.IsSettingsSceneLoaded == false)
```

Comment:
Not a requirement just something to note. This can be written shorthand as
if (!AframaxSceneManager.Instance.IsSettingsSceneLoaded)
! means not